pdsfile/pds4file/rules/uranus_occs_earthbased.py

```diff
@@ -52,131 +52,41 @@
+         r'bundles/\1/browse\3.xml',
+        ]),
+    (r'documents/uranus_occs_earthbased.*', 0,
+        [r'bundles/uranus_occs_earthbased',
+         r'bundles/uranus_occs_earthbased',
+         r'bundles/uranus_occs_earthbased',
+        ]),
 ])
 
 associations_to_previews = translator.TranslatorByRegex([
-
-    # COISS_1xxx and COISS_2xxx
-    (r'.*/(COISS_[12]xxx)(|_v[0-9\.]+)/(COISS_....)/(data|extras/\w+)/(\w+/[NW][0-9]{10}_[0-9]+).*', 0,
-            [r'previews/\1/\3/data/\5_full.png',
-             r'previews/\1/\3/data/\5_med.jpg',
-             r'previews/\1/\3/data/\5_small.jpg',
-             r'previews/\1/\3/data/\5_thumb.jpg',
-            ]),
-    (r'.*/(COISS_[12]xxx)(|_v[0-9\.]+)/(COISS_....)/(data|extras/\w+)(|/\w+)', 0,
-            r'previews/\1/\3/data\5'),
-    (r'.*/(COISS_[12])999.*', 0,
-            r'previews/\1xxx'),
-
-    # COISS_3xxx
-    (r'.*/(COISS_3xxx.*/COISS_3...)/(data|extras/\w+)/(images/\w+[A-Z]+)(|_[a-z]+)\..*', 0,
-            [r'previews/\1/data/\3_full.jpg',
-             r'previews/\1/data/\3_med.jpg',
-             r'previews/\1/data/\3_small.jpg',
-             r'previews/\1/data/\3_thumb.jpg',
-            ]),
-    (r'.*/(COISS_3xxx.*/COISS_3...)/(data|extras/\w+)/(maps/\w+_SMN).*', 0,
-            [r'previews/\1/data/\3_full.png',
-             r'previews/\1/data/\3_med.png',
-             r'previews/\1/data/\3_small.png',
-             r'previews/\1/data/\3_thumb.png',
-            ]),
-    (r'.*/(COISS_3xxx.*/COISS_3...)/(data|extras/\w+)(|/images|/maps)', 0,
-            [r'previews/\1/data/\3',
-             r'previews/\1/extras/browse/\3',
-             r'previews/\1/extras/thumbnail/\3',
-             r'previews/\1/extras/full/\3',
-            ]),
-    (r'.*/(COISS_3xxx.*/COISS_3...)/extras', 0,
-            r'previews/\1/data'),
+    (r'.*/(uranus_occs_earthbased/uranus_occ_u.*/(data|browse)(.*|_[a-z]*])/.*)\.[a-z]{3}', 0,
+        [r'previews/\1_full.png',
+         r'previews/\1_med.png',
+         r'previews/\1_small.png',
+         r'previews/\1_thumb.png',
+        ])
 ])
 
 associations_to_metadata = translator.TranslatorByRegex([
-    (r'.*/(COISS_[12]xxx)(|_v[0-9\.]+)/(COISS_....)/(data|extras/w+)/\w+/([NW][0-9]{10}_[0-9]+).*', 0,
-            [r'metadata/\1/\3/\3_index.tab/\5',
-             r'metadata/\1/\3/\3_ring_summary.tab/\5',
-             r'metadata/\1/\3/\3_moon_summary.tab/\5',
-             r'metadata/\1/\3/\3_saturn_summary.tab/\5',
-             r'metadata/\1/\3/\3_jupiter_summary.tab/\5',
-            ]),
-    (r'metadata/(COISS_.xxx/COISS_[12])...', 0,
-            r'metadata/\g<1>999'),
-    (r'metadata/(COISS_.xxx/COISS_[12]).../(COISS_.)..._(.*)\..*', 0,
-            [r'metadata/\g<1>999/\g<2>999_\3.tab',
-             r'metadata/\g<1>999/\g<2>999_\3.csv',
-             r'metadata/\g<1>999/\g<2>999_\3.lbl',
-            ]),
+    (r'.*/(uranus_occs_earthbased)/(uranus_occ_u.*)/(data|browse)(.*|_[a-z]*])/(rings|global)/(.*)\.[a-z]{3}', 0,
```

Comment:
There will only be one metadata directory for the whole bundleset, not a subdirectory per bundle.